helm-charts/basehub/values.yaml

```diff
@@ -206,6 +206,9 @@ jupyterhub:
             # this is what allows us to shutdown servers
             # when people leave a notebook open and wander off
             cull_connected: true
+          NotebookApp:
+            extra_template_paths:
+              - /usr/local/share/jupyter/custom_template
```

Comment:
I'm setting this one in the `basehub` template because it does no harm if we set it here and we provide no custom templates.

The benefit of this being here is that we can inject `extraFiles` for other hubs too to extend the templates if need be in the future.

Or at least until will decide on a better strategy as part of https://github.com/2i2c-org/infrastructure/issues/1697 that might imply sync-ing the templates from a repo on the fly like we do with the hub templates.